.drone.star

```diff
@@ -800,6 +811,7 @@ def acceptance(ctx):
         "oc10IntegrationAppIncluded": False,
         "skip": False,
         "debugSuites": [],
+        "earlyFail": False,
```

Comment:
what about setting the default to `True`

Reply:
fixed